nss-day-cohort-30/bangazon-site-plucky-pikes
# seeded database

# Description
Seeded database.

## Type of change
Please delete options that are not relevant.


# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout initial-setup`
1. run Visual Studio Code
1. run Azure Data Studio and connect to BangazonSite to make sure tables are included


# Checklist:
